src/datasets/basics.jl

```diff
@@ -93,8 +93,20 @@ function load_data(dataset::String, ratios=(0.6,0.2,0.2); seed=nothing,
 		data_normal, data_anomalous = load_uci_data(dataset; kwargs...)
 	elseif dataset in mldatasets # MNIST,FMNIST, SVHN2, CIFAR10
 		data_normal, data_anomalous = load_mldatasets_data(dataset; kwargs...)
-	elseif dataset in other_datasets
+	elseif dataset in other_datasets # other tabular datasets
 		data_normal, data_anomalous = load_other_data(dataset; standardize=true, kwargs...)
+	elseif dataset =="MNIST-C"
+		data_normal, data_anomalous = load_mnist_c_data(; kwargs...)
+	elseif dataset == "MVTec-AD"
+		data_normal, data_anomalous = load_mvtec_ad_data(; kwargs...)
+	elseif occursin("MNIST-C", dataset)
+		sd = split(dataset, "_")
+		category = (length(sd) >= 2) ? dataset[9:end] : "brightness"
```

Comment:
Add comment as to why this case has to be handled separately.

Reply:
```julia
dataset = "MNIST-C_fog"
sd = split(dataset, "_")
category = (length(sd) >= 2) ? dataset[9:end] : "brightness"
```
`category` now evaluates to `"fog"`